packages/react/src/InlineMessage/InlineMessage.dev.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import type {Meta} from '@storybook/react'
+import React from 'react'
+import {InlineMessage} from '.'
+
+const meta = {
+  title: 'Experimental/Components/InlineMessage/Dev',
+  component: InlineMessage,
+} satisfies Meta<typeof InlineMessage>
+
+export default meta
+
+export const Default = () => {
```

Comment:
One note with what Katie shared earlier, we should make sure these get added to our e2e tests too as we're adding in dev stories